apps/condo/domains/organization/integrations/sbbol/SbbolFintechApi.js

```diff
@@ -230,6 +365,73 @@ class SbbolFintechApi extends SbbolRequestApi {
     get apiPrefix () {
         return '/fintech/api'
     }
+
+    /**
+     * Get extended client info request, posted with `getClientInfoRequest`
+     *
+     * @return {Promise<{error: any}|{ClientInfo: responseClientInfo}>}
+     */
+    async getClientInfo () {
+        const { data, statusCode } = await this.request({
+            method: 'GET',
+            path: this.getClientInfoRequestPath,
+        })
+
+        try {
+            const jsonData = JSON.parse(data)
+            const error = get(jsonData, 'error')
+
+            if (error) return logger.error({ msg: 'getClientInfo from SBBOL error', error })
+
+            if (statusCode === 200) {
+                return { ClientInfo: jsonData }
```

Comment:
Maybe just `data`? Entities starting from upper case can be misunderstood as classes

```suggestion
                return { data: jsonData }
```